### ISSUE TITLE
How to change the boot partition?

### ISSUE THREAD

**Issue by tobiasBora** [nix-community/raspberry-pi-nix#61]
In order to have a bit of fun I wanted to try ZFS on the pi5 to get snapshots, incremental backups etc. so I got the kernel working, but now I need to update the configuration to make it work, and my main question is on how to setup the `/boot` partition. Of course, it cannot be a ZFS partition, so I need a separate FAT32 partitition, but I don't know, can I just assume that the FIRMWARE partition contains the /boot partition and don't do anything else beside adding a bunch of
```
  fileSystems."/home" =
    { device = "zfs_pool/home";
      fsType = "zfs";
    };
```
in my `configuration.nix`? Or do I need more care here?

**Comment by tstat** [comment]
I don't think you need more care than that, but don't know for certain. If you try it out please report back how it goes.
